third-party/realdds/src/dds-device-server.cpp

```diff
@@ -41,17 +44,34 @@ dds_device_server::~dds_device_server()
 }
 
 
-void dds_device_server::start_streaming( const std::string & stream_name,
-                                         const image_header & header )
+void dds_device_server::start_streaming( const std::vector< std::pair < std::string, image_header > > & streams_to_open )
```

Comment:
As I said in our discussion:
I think this should not be a command (to the server or the stream) but rather the direct result of someone subscribing to the stream. The application needs to be notified when this happens (because it does the streaming).
Not for this PR.

Reply:
Let's speak about it later on.
I think it's better to stick to USB style use case.
Open --> start stream
Start --> connect a callback

Reply:
>I think it's better to stick to USB style use case.
Not sure I agree.
There's no meaning to providing a callback in DDS, therefore `start` is meaningless. And `open` is just too generic.
As I said elsewhere:
I don't think there should be an `open` or `start_streaming` at all: this is DDS. When someone subscribes to a stream, the server should automatically start given the set profile. Same with stopping.